build.cake

```diff
@@ -412,108 +413,44 @@ Task("PackageChocolatey")
 	.Description("Creates chocolatey packages of the console runner")
```

Comment:
I'll review that. The objective is for Package to be dependent but not RePackage.